custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch/attributes/default.rb

```diff
@@ -0,0 +1,40 @@
+default['elasticsearch'].tap do |elasticsearch|
+  # Run Elasticsearch on util instances containing elasticsearch in name
+  # This is the default
+  elasticsearch['instance_name'] = 'elasticsearch'
+  elasticsearch['is_elasticsearch_instance'] = (
+    node['dna']['instance_role'] == 'util' &&
+    node['dna']['name'].include?(elasticsearch['instance_name'])
+  )
+
+  # Run Elasticsearch on a solo or app_master instance
+  # Not recommended for production environments
+  #elasticsearch['is_elasticsearch_instance'] = ( ['solo', 'app_master'].include?(node['dna']['instance_role']) )
+
+  # Elasticsearch version to install
+  # Go to https://www.elastic.co/downloads/past-releases to see the available version
+  # elasticsearch['version'] = '5.6.15'
+  elasticsearch['version'] = 6.6.3'
```

Comment:
@rupikakapoor  should it be 

```rb
elasticsearch['version'] = '6.6.3'
```

?

lacking a `'` before 6.6.3

Reply:
Thank you, this is fixed now